docker/docker-compose.yaml

```diff
@@ -164,7 +164,6 @@ services:
 
 networks:
   von:
-    external:
```

Comment:
?

Reply:
sorry missed off adding `external: true`, docker v3 has different format for `networks`
https://docs.docker.com/compose/compose-file/compose-file-v3/#network-configuration-reference